jsonnet/kube-prometheus/components/grafana.libsonnet

```diff
@@ -54,10 +56,17 @@ function(params) {
         rawDashboards: g.config.rawDashboards,
         folderDashboards: g.config.folderDashboards,
         containers: g.config.containers,
-      },
+        config+: g.config.config,
+      } + (
+        // conditionally overwrite default setting
+        if std.length(g.config.datasources) > 0 then
+        { datasources: g.config.datasources }
+        else {}
+      )
     },
   },
 
+  config: glib.grafana.config,
```

Comment:
I will take care of those mapping after I finish testing kube-prometheus inclusion with https://github.com/openshift/cluster-monitoring-operator/pull/1044